server/src/migrations/Migration20210220195842.ts

```diff
@@ -0,0 +1,12 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20210220195842 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('create table "event" ("id" varchar(255) not null, "created_at" timestamptz(0) not null, "updated_at" timestamptz(0) not null, "name" varchar(255) not null, "description" varchar(255) not null);');
+    this.addSql('alter table "event" add constraint "event_pkey" primary key ("id");');
+
+    this.addSql('alter table "club" drop column "is_deleted";');
```

Comment:
Why are we altering the `club` table here?

Reply:
This fixes the "is_deleted" column under clubs that was accidentally added from the get-all-clubs PR. Making a new PR to clean up the clubs migration mistake.